### ISSUE TITLE
Mat stepper support

### ISSUE THREAD

**Issue by nitin27may** [dschnelldavis/angular2-json-schema-form#123]
Hi,

We need support for the mat stepper, Can we get support for step based form.

**Comment by ChrisTUth** [comment]
i do not think this is possible until #120 is fixed. because the steppers load all steps.

**Comment by nitin27may** [comment]
Could you please add more details ? What error you are getting ?
